6_commands.ipynb

```diff
@@ -11,6 +13,19 @@
     "</div> "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Convenient commands\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Objective</b>\n",
+    "    <p>This notebook will collect convenient commands to help you.</p>\n",
```

Comment:
What do you think about this:

    "    <p>In this notebook, we will collect convenient commands to help you.</p>\n",

Reply:
"dictionary of utility commands" or "glossary of utility commands" may be appropriate here.

Reply:
"glossary of utility commands" feels quite nice. :)
:+1: